### PR TITLE
nv_gpu_install_from_commit: update builder script for new GPU Operator repo structure

### PR DESCRIPTION
**Pull request by kpouget**

### CODE REVIEW

File: roles/nv_gpu_install_from_commit/files/operator_helper_image_builder.yml
Comment (ArangoGutierrez) [inline roles/nv_gpu_install_from_commit/files/operator_helper_image_builder.yml:54]: This one is interesting

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm
/hold

File: roles/nv_gpu_install_from_commit/files/operator_image_builder_pod.yml
Comment (ArangoGutierrez) [inline roles/nv_gpu_install_from_commit/files/operator_image_builder_pod.yml:28]: 4.8 is 1.16, maybe we want to consider it

Comment (kpouget) [reply roles/nv_gpu_install_from_commit/files/operator_image_builder_pod.yml:28]: hum, I don't really know, I just used whatever they use upstream,
what do you thing?

Comment (ArangoGutierrez) [reply roles/nv_gpu_install_from_commit/files/operator_image_builder_pod.yml:28]: 1.15 is ok for now, 4.8 is 2 months in the horizon. we can bump to 1.16 later on

**Comment by kpouget** [comment]
`ci/prow/gpu-operator-e2e ` fails because the master branch of the GPU Operator is currently broken.
the fix should be merged today, I'll wait for it before merging this PR

/hold

**Comment by kpouget** [comment]
\test gpu-operator-e2e

```
2021-03-25 17:50:09,249 p=3643 u=psap-ci-runner n=ansible | TASK: nv_gpu : Ensure that the GPU Operator PackageManifest exists
2021-03-25 17:50:09,250 p=3643 u=psap-ci-runner n=ansible | ----- FAILED ----
2021-03-25 17:50:09,250 p=3643 u=psap-ci-runner n=ansible | msg: non-zero return code
2021-03-25 17:50:09,250 p=3643 u=psap-ci-runner n=ansible | 
2021-03-25 17:50:09,250 p=3643 u=psap-ci-runner n=ansible | <command> oc get packagemanifests/gpu-operator-certified -n openshift-marketplace
2021-03-25 17:50:09,250 p=3643 u=psap-ci-runner n=ansible | 
2021-03-25 17:50:09,250 p=3643 u=psap-ci-runner n=ansible | <stderr> Error from server (NotFound): packagemanifests.packages.operators.coreos.com "gpu-operator-certified" not found
2021-03-25 17:50:09,250 p=3643 u=psap-ci-runner n=ansible | ----- FAILED ----
```
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/101/pull-ci-openshift-psap-ci-artifacts-master-gpu-operator-e2e/1375124095917625344/artifacts/gpu-operator-e2e/presubmit-operatorhub/artifacts/175004_gpu-operator__deploy_from_operatorhub.sh/ansible_logs

File: roles/nv_gpu_install_from_commit/files/operator_helper_image_builder.yml
Comment (kpouget) [reply roles/nv_gpu_install_from_commit/files/operator_helper_image_builder.yml:54]: `go` is called inside the `Makefile`, I install it to mute a warning at the beginning of `make docker-build`

**Comment by kpouget** [comment]
the GPU Operator `master` branch is still broken, even when trying the PR that was supposed to fix it:

```
NAME                                 DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR                                  AGE
nvidia-container-toolkit-daemonset   1         1         1       1            1           nvidia.com/gpu.deploy.container-toolkit=true   15m
nvidia-device-plugin-daemonset       1         1         0       1            0           nvidia.com/gpu.deploy.device-plugin=true       15m
nvidia-driver-daemonset              1         1         1       1            1           nvidia.com/gpu.deploy.driver=true              15m
```

**Comment by kpouget** [comment]
the PR was `lgtm`-ed by Eduardo and the patch successfully deploys the GPU Operator from its `master` branch, 
so I force merge as the `gpu-operator-e2e` test won't pass until the issue is fixed upstream.

**Comment by ArangoGutierrez** [comment]
Merged!

**Pull request merged**
